app/openfoodfacts.py

```diff
@@ -303,6 +306,57 @@ def select_nutriments(self, document: JSONType):
             ):
                 nutriments.pop(key)
 
+    def transform_nova_groups_markers(self, document: JSONType):
+        """Transform Nova Groups markers into a list"""
+        if "nova_groups_markers" not in document:
+            return
+        nova_groups_markers = document["nova_groups_markers"]
+        document["nova_groups_markers"] = [
+            {
+                "id": nova_group,
+                "marker": [{"type": marker[0], "id": marker[1]} for marker in markers],
+            }
+            for nova_group, markers in nova_groups_markers.items()
+        ]
+
+    def transform_images(self, document: JSONType):
+        if "images" not in document:
+            return
+        uploaded = document["images"]["uploaded"]
+        # create uploaded_images
+        document["uploaded_images"] = [
+            {
+                "id": key,
+                "uploaded_t": image["uploaded_t"],
+                "uploader": image["uploader"],
+                "full_size": {"h": image["h"], "w": image["w"]},
+            }
+            for key, image in uploaded.items()
+        ]
+        # selected images
+        document["selected_images"] = {}
+        for image_type, selected in document["images"]["selected"].items():
```

Comment:
```suggestion
        for image_type, selected in document["images"].get("selected", {}).items():
```
the `selected` field can be missing from the `images`, I got the issue on Robotoff after the new `images` schema was deployed in production.